src/app/screens/LNURLOpenChannel.tsx

```diff
@@ -0,0 +1,119 @@
+import ConfirmOrCancel from "@components/ConfirmOrCancel";
+import Container from "@components/Container";
+import PublisherCard from "@components/PublisherCard";
+import SuccessMessage from "@components/SuccessMessage";
+import axios from "axios";
+import { useState, MouseEvent } from "react";
+import { toast } from "react-toastify";
+import { USER_REJECTED_ERROR } from "~/common/constants";
+import api from "~/common/lib/api";
+import msg from "~/common/lib/msg";
+import getOriginData from "~/extension/content-script/originData";
+import type { LNURLOpenChannelServiceResponse } from "~/types";
+
+type Props = {
+  details: LNURLOpenChannelServiceResponse;
+  origin?: {
+    name: string;
+    icon: string;
+  };
+};
+
+function LNURLOpenChannel(props: Props) {
+  const [origin] = useState(props.origin || getOriginData());
+  const { uri } = props.details;
+  const [pubkey, host] = uri.split("@");
+
+  const [loadingConfirm, setLoadingConfirm] = useState(false);
+  const [successMessage, setSuccessMessage] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  async function confirm() {
+    try {
+      setErrorMessage("");
+      setLoadingConfirm(true);
+      await api.connectPeer({
+        host,
+        pubkey,
+      });
+      // todo: check success
```

Comment:
This is outdated, yes?

Reply:
+1 looks like 

though we do not check if the `connectPeer` call was successful?

Reply:
No, but doesn't this take care of it?
https://github.com/getAlby/lightning-browser-extension/pull/1173/files#diff-75c67e9b6d920ae59f2bb6ec2011595392fa96986711b39bb1e57f36ded8f5ceR21-R28

Reply:
ah ok, we have that global `try {` where we catch that error. 🙈